continuous_integration/recipe/meta.yaml

```diff
@@ -30,6 +30,7 @@ requirements:
     - setuptools-rust>=1.1.2
   run:
     - python
+    - setuptools-rust>=1.1.2
```

Comment:
Based on the discussion [here](https://github.com/dask-contrib/dask-sql/pull/471#discussion_r853217325) is this needed?

Reply:
No, it is not needed. I had based this PR off the `datafusion-aggregate` one and probably did that before removing this in the other PR. It can be removed and I will remove it.